sample/kotlin/src/main/res/layout/fragment_otel_traces.xml

```diff
@@ -82,6 +82,16 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintTop_toBottomOf="@id/linked_spans_test"/>
 
+    <androidx.appcompat.widget.AppCompatButton
+        android:id="@+id/start_rust_tracer_stress_test"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_margin="16dp"
+        android:text="Start Rust Tracer Stress Test"
```

Comment:
prefer to use @string instead of providing text directly. Such code will add warning by the IDE. Lot of warning writhing one file makes issues detection harder